openapi2beans/pkg/generator/yaml2java_test.go

```diff
@@ -0,0 +1,892 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package generator
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/galasa-dev/cli/pkg/files"
+	"github.com/stretchr/testify/assert"
+)
+
+func getGeneratedCodeFilePathWithPackage(storeFilepath string, packageName string, name string) string {
+	return storeFilepath + "/src/main/java/" + packageName + "/" + name + ".java"
```

Comment:
Don't add the src/main/java onto the path please. It causes the code to come out in the wrong place.
It assumes that everyone using the tool uses the maven coordinates.

Also, aren't you deleting the storeFilePath ? Perhaps you shouldn't... as it's dangerous. Despite what I said earlier.